### PR TITLE
Skill modification (according to the official server setting)

### PR DESCRIPTION
**Pull request by cjlaaa**
Skill modification (according to the official server setting)
1.The skill ProtectionField MinAC increase value modified.
2.The skill Rage MinDC increase value modified.
3.The skill Plague's damage should be the MaxSC * 2.

### CODE REVIEW

File: Server/MirObjects/PlayerObject.cs [hunk 1]
```diff
@@ -7684,7 +7684,7 @@ private void Rage(UserMagic magic)
 
             int duration = 18 + (6 * magic.Level);
             int maxDC = (int)Math.Round(Stats[Stat.MaxDC] * (0.12 + (0.03 * magic.Level)));
-            int minDC = (int)Math.Round(Stats[Stat.MinDC] * (0.12 + (0.03 * magic.Level)));
+            int minDC = (int)Math.Round(Stats[Stat.MaxDC] * (0.12 + (0.03 * magic.Level)));
```
Comment (Suprcode) [inline Server/MirObjects/PlayerObject.cs:7687]: Why is minDC variable getting MaxDC stat?

File: Server/MirObjects/PlayerObject.cs [hunk 2]
```diff
@@ -7672,7 +7672,7 @@ private void ProtectionField(UserMagic magic)
 
             int duration = 45 + (15 * magic.Level);
             int maxAC = (int)Math.Round(Stats[Stat.MaxAC] * (0.2 + (0.03 * magic.Level)));
-            int minAC = (int)Math.Round(Stats[Stat.MinAC] * (0.2 + (0.03 * magic.Level)));
+            int minAC = (int)Math.Round(Stats[Stat.MaxAC] * (0.2 + (0.03 * magic.Level)));
```
Comment (Suprcode) [inline Server/MirObjects/PlayerObject.cs:7675]: Why is minAC getting MaxAC stat?

File: Server/MirObjects/PlayerObject.cs (hunk 1 above)
Comment (cjlaaa) [reply Server/MirObjects/PlayerObject.cs]: The skill ProtectionField's MinDC increase should be the same as MaxDC.
The code has been changed to a better way.

File: Server/MirObjects/PlayerObject.cs (hunk 2 above)
Comment (cjlaaa) [reply Server/MirObjects/PlayerObject.cs]: The skill Rage's MinAC increase should be the same as MaxAC.
The code has been changed to a better way.

**Comment by cjlaaa** [comment]
The skill ProtectionField's MinDC increase should be the same as MaxDC.
The skill Rage's MinAC increase should be the same as MaxAC.
The code has been changed to a better way.

ProtectionField的MinDC增加值应该和MaxDC的增加值相同.
Rage的MinAC增加值应该和MaxAC的增加值相同.
代码已改成更好的实现方式.

**Pull request merged**